Orange/widgets/visualize/owpythagorastree.py

```diff
@@ -40,6 +40,7 @@
 
 class OWPythagorasTree(OWWidget):
     name = 'Pythagorean Tree'
+    short_name = 'Pythagora\nTree'
```

Comment:
I think this short form doesn't function in English. It would be Pythagoras, and if you randomly cut the last two letters you get Pythagore. I would say Pythagor or Pythagoras.